### PR TITLE
don't double report errors

### PR DESCRIPTION
**Pull request by snopoke**
## Technical Summary
Avoid double reporting errors to Sentry from web apps. I've currently only applied this to errors during form sessions but I think there are likely others we could apply this too as well.

### Safety story
Small change that should only affect error reporting. I tested a few different scenarios locally without issue.

### Automated test coverage
None that I'm aware of

### QA Plan
I don't think this needs QA but I'm open to change that.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by snopoke** [comment]
JS build failing: " 1) Should handle timeout error".

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js [hunk 1]
```diff
@@ -180,9 +180,9 @@ hqDefine("cloudcare/js/formplayer/app", function () {
         data.onerror = function (resp) {
             var message = resp.human_readable_message || resp.exception || "Unexpected Error";
```
Comment (orangejenny) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js:181]:
```suggestion
            var message = resp.human_readable_message || resp.exception || gettext("Unexpected Error");
```

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js [hunk 2]
```diff
@@ -180,9 +180,9 @@ hqDefine("cloudcare/js/formplayer/app", function () {
...
             if (resp.is_html) {
-                showHTMLError(message, $("#cloudcare-notifications"));
+                showHTMLError(message, $("#cloudcare-notifications"), resp.reportToHq);
```
Comment (orangejenny) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js:183]: Doesn't this line, and the one below, need to pass `undefined` for `autoHideTime`?

Comment (snopoke) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js:183]: Functions in JS are just weird. There are two things specifically that make this OK:
1. Function arguments default to `undefined` so they don't need to be provided (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions#default_parameters)
2. Functions can accept any number of arguments regardless of what the function definition is.

Comment (orangejenny) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js:183]: I'm still not quite following - `showHTMLError` has the param list `message, $el, autoHideTime, reportToHq`, so won't that interpret `resp.reportToHq` as `autoHideTime`, and `reportToHq` will always be undefined?

Comment (snopoke) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js:183]: 🤦🏻‍♂️ yes (but only for `showHTMLError` and not `showError`): 4ca7face7ab

**Review by orangejenny — APPROVED** [review]

**Pull request merged**